From e833f378a0a15ba22a658eaa5bf11968bd8292b7 Mon Sep 17 00:00:00 2001
From: Jiyong Park <jiyong@google.com>
Date: Mon, 7 Aug 2017 13:15:50 +0900
Subject: [PATCH 19/47] Make software codecs as VNDK

Software codecs and their dependencies are marked as VNDK (or just
vendor_available:true for static/header libs).

Bug: 37343418
Test: build the software codecs with BOARD_VNDK_VERSION=current
Change-Id: I2aecebfc5d1e5cc1adb804069964303996410070
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index e5e11f8..df2e2b9 100644
--- a/Android.bp
+++ b/Android.bp
@@ -2072,6 +2072,7 @@ gensrcs {
 
 cc_library_static {
     name: "libvpx",
+    vendor_available: true,
 
     arch: {
         arm: {
-- 
2.17.1

